apps/website/src/content/docs/notificationmarker.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: NotificationMarker
+description: Shows a notification marker dot at the top right of the component.
+propsPath: '@itwin/itwinui-react/esm/core/NotificationMarker/NotificationMarker.d.ts'
+thumbnail:
+---
+
+<p>{frontmatter.description}</p>
+
+<LiveExample src='NotificationMarker.main.jsx'>
+  <AllExamples.NotificationMarkerMainExample client:load />
+</LiveExample>
+
+## Usage
+
+### Status
```

Comment:
I had left it blank since I noticed this pattern in other pages too: [Input](https://dev-itwinui.bentley.com/docs/input#usage), [ToggleSwitch](https://dev-itwinui.bentley.com/docs/toggleswitch#usage), [Breadcrumbs](https://dev-itwinui.bentley.com/docs/breadcrumbs#usage). So, I guess this needs to be fixed for those pages as well. Added a point for it in the maintenance issue ([#1571](https://redirect.github.com/iTwin/iTwinUI/issues/1571)).

For this PR, I fixed it in [`55d01b1` (#1999)](https://github.com/iTwin/iTwinUI/pull/1999/commits/55d01b15c7d866fc29edd6d72b7373585e4b35ae).

Reply:
Would be better suited for #961 or #941

Reply:
I considered those but thought that [#961](https://redirect.github.com/iTwin/iTwinUI/issues/961) was to add docs for each component and [#941](https://redirect.github.com/iTwin/iTwinUI/issues/941) was more about website features. So, I had chosen [#1571](https://redirect.github.com/iTwin/iTwinUI/issues/961).

Moved it now to [#941](https://redirect.github.com/iTwin/iTwinUI/issues/941).